### PR TITLE
Notification panel foundation

### PR DESCRIPTION
**Pull request by taleldayekh**
**Thank you for submitting this pull request :)**

Fixes # 

**Short description**
Adds the foundation for a notification panel. A new modal with a separate routing which displays the notification panel.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Open the dApp and connect
2. When connected a notification icon will appear in the header from which you can access the notification panel.

### CODE REVIEW

File: raiden-dapp/src/components/notification-panel/NotificationCard.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </div>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({})
+export default class NotificationCard extends Vue {}
```
Comment (kelsos) [inline raiden-dapp/src/components/notification-panel/NotificationCard.vue:22]: I am guessing we are not adding the properties as part of this PR? right

File: raiden-dapp/src/views/NotificationPanel.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<template>
+  <div id="notification-panel-wrapper">
```
Comment (kelsos) [inline raiden-dapp/src/views/NotificationPanel.vue:2]: could we make this `class="notification-panel"` and the other class maybe `class="notification-panel__content"` ?

File: raiden-dapp/src/components/notification-panel/NotificationCard.vue (hunk 1 above)
Comment (taleldayekh) [reply raiden-dapp/src/components/notification-panel/NotificationCard.vue:22]: No, we'll add them with the first notification task. :)

File: raiden-dapp/src/views/NotificationPanel.vue (hunk 2 above)
Comment (taleldayekh) [reply raiden-dapp/src/views/NotificationPanel.vue:2]: Yes sir!

File: raiden-dapp/src/components/notification-panel/NotificationCard.vue (hunk 1 above)
Comment (kelsos) [reply raiden-dapp/src/components/notification-panel/NotificationCard.vue:22]: Ok sounds good :+1:

File: raiden-dapp/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 ### Added
 
+- [#168] Notification panel
```
Comment (kelsos) [inline raiden-dapp/CHANGELOG.md:9]: Same comment Thomas made on my PR, can you move that under `[Unreleased]`

**Pull request merged**